config/200-controller-clusterrole.yaml

```diff
@@ -76,6 +76,16 @@ rules:
     verbs:
       - "update"
 
+  # Our own resources and statuses we care about.
+  - apiGroups:
+      - "messaging.knative.dev"
+    resources:
+      - "pipelines"
+      - "pipelines/status"
+      - "inmemorychannels"
```

Comment:
This will need to be an aggregated role. Similar to #1341, but include all permissions, not just read and update.